internal/operators/lvm/lvm_operator.go

```diff
@@ -57,6 +61,18 @@ func (o *operator) GetFullName() string {
 	return "Logical Volume Management"
 }
 
+func (o *operator) GetStorageClassName() string {
```

Comment:
Small nit: Getter functions don't need the word "Get" in for golang https://go.dev/doc/effective_go#Getters